app/api/private-cloud/products/[licencePlate]/comments/route.ts

```diff
@@ -28,7 +29,27 @@ export const POST = createApiHandler({
   return CreatedResponse(comment);
 });
 
-export async function GET() {
-  const data = await listOp();
-  return NextResponse.json(data);
-}
+const PathParamsSchema = z.object({
+  licencePlate: z.string(),
+});
+
+export const GET = createApiHandler({
+  roles: ['user'],
+  permissions: [PermissionsEnum.ViewAllPrivateProductComments],
+  validations: {
+    pathParams: PathParamsSchema,
+  },
+})(async ({ session, pathParams }) => {
+  if (!session) {
```

Comment:
session check is not necessary as we checks roles & permissions within the api handler.